odbc/src/api/functions.rs

```diff
@@ -375,21 +409,57 @@ pub extern "C" fn SQLDriverConnect(
     _string_length_2: *mut SmallInt,
     _drive_completion: DriverConnectOption,
 ) -> SqlReturn {
+    dbg!();
     unsupported_function(MongoHandleRef::from(connection_handle), "SQLDriverConnect")
 }
 
 #[no_mangle]
 pub extern "C" fn SQLDriverConnectW(
-    _connection_handle: HDbc,
+    connection_handle: HDbc,
     _window_handle: HWnd,
-    _in_connection_string: *const WChar,
-    _string_length_1: SmallInt,
+    in_connection_string: *const WChar,
+    string_length_1: SmallInt,
     _out_connection_string: *mut WChar,
```

Comment:
I looked into the spec and it seems that we should copy *InConnectionString to *OutConnectionString because we are not planning to support SQL_DRIVER_PROMPT (see [Driver Guidelines](https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqldriverconnect-function?view=sql-server-ver16#driver-guidelines)).